tests_integration/nativeapp/test_large_upload.py

```diff
@@ -55,7 +55,7 @@ def mock_getenv(key: str, default: str | None = None) -> str | None:
             yield
 
 
-@pytest.mark.skip  # FIXME: requires AWS + python connector to support threshold=<number>
+# @pytest.mark.skip  # FIXME: requires AWS + python connector to support threshold=<number>
```

Comment:
i don't think this meant to make it in

Reply:
Yeah it was, I was paranoid that I broke the test so I unskipped it for now!